(DOCSP-12904): Link to Supplementary Examples - iOS SDK (#941)
mongodb/docs-realm: source/sdk/ios/additional-examples.txt

.. _ios-additional-examples:

=============================
Additional Examples - iOS SDK
=============================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

Overview
--------

> The Realm community has created many applications that demonstrate the usage of the Realm iOS SDK. This page contains some of these applications, and the MongoDB documentation team will update this page with additional examples over time.

Reviewer: Always use the source constants for the product name.
```suggestion
The {+service-short+} community has created many applications that demonstrate the usage of the Realm iOS SDK. This page contains some of these applications, and the MongoDB documentation team will update this page with additional examples over time.
```

> The Realm community has created many applications that demonstrate the usage of the Realm iOS SDK. This page contains some of these applications, and the MongoDB documentation team will update this page with additional examples over time.

Reviewer: In the note below, you mention the "MongoDB Documentation team", but in this sentence, you write it as "MongoDB documentation team" (no capital 'D' on "documentation"). We should keep this identifier consistent.

Author: fixed 👍

> The Realm community has created many applications that demonstrate the usage of the Realm iOS SDK. This page contains some of these applications, and the MongoDB documentation team will update this page with additional examples over time.

Reviewer: Can we cut off this line at 80 characters? It's super long.

Author: Fixed

.. note:: 

   The MongoDB Documentation team does not directly maintain these examples. 

Examples by the Realm Engineering Team
--------------------------------------
You can find :github:`example applications
<realm/realm-cocoa/tree/master/examples>` for both iOS and macOS that
demonstrate how to use many Realm features, such as migrations, encryption, use

> of {+realm+} with ``UITableViewController``, and more. 

Reviewer: Since "use of realm with UITableViewController" isn't a "Realm feature" as you introduce this list, maybe you could rewrite this into two sentences? Also, be sure to use the {+service-short+} source constant for all mentions of Realm in paragraph text.

```
The :github:`realm-cocoa example applications
<realm/realm-cocoa/tree/master/examples>` provide examples for both iOS and macOS that
demonstrate how to use {+service-short+} features such as migrations, encryption,
and {+sync+}. The examples also demonstrate how to combine {+service-short+} with
common iOS concepts like ``UITableViewController``. 
```

Author: Yup that sounds good, copied almost verbatim except I changed "... features such as migrations, encryption,
and {+sync+}" to remove Realm Sync, since the iOS examples in that project do not contain Realm Sync examples.

.. _ios-additional-examples:

=============================
Additional Examples - iOS SDK
=============================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

Overview
--------

> There are many examples of applications that the {+realm+} community has

Reviewer: ```suggestion
There are many examples of applications that the {+service-short+} community has
```
The "realm" substitution is just for the database/data structure, not the product or community.

Reviewer: The introductory sentence reads a bit awkwardly to me. I wonder if we could rephrase it to something like "Every day, the MongoDB community creates additional applications that demonstrate {+service-short+}." Or maybe that is too marketing-y?

created. This page contains some of these applications, and the MongoDB documentation
team will update this page with additional examples over time.

.. note:: 

   The MongoDB Documentation team does not directly maintain these examples. 

Examples by the Realm Engineering Team
--------------------------------------
You can find :github:`example applications
<realm/realm-cocoa/tree/master/examples>` for both iOS and macOS that
demonstrate how to use many Realm features, such as migrations, encryption, use

> of {+realm+} with UITableViewControllers, and more. 

Reviewer: ```suggestion
of {+realm+} with ``UITableViewController``, and more. 
```
Should this be monospaced?

.. _ios-additional-examples:

=============================
Additional Examples - iOS SDK
=============================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

Overview
--------
The {+service-short+} community has created many applications that demonstrate
the usage of the Realm iOS SDK. This page contains some of these applications,
and the MongoDB Documentation team will update this page with additional
examples over time.

.. note:: 

   The MongoDB Documentation team does not directly maintain these examples. 

Examples by the Realm Engineering Team
--------------------------------------
The :github:`realm-cocoa example applications
<realm/realm-cocoa/tree/master/examples>` provide examples for both iOS and

> macOS that demonstrate how to use {+service-short+} features such as migrations,

Reviewer: unneeded comma
```suggestion
macOS that demonstrate how to use {+service-short+} features such as migrations
```
